test/fixtures/ncbi_database_truncator.py

```diff
@@ -67,7 +71,7 @@
     rqst = cur.fetchall()
     for ind in range(0, len(rqst)):
         ins = dest_db.execute(
-            "insert into names values (" + ",".join(["?"] * 3) + ")", rqst[ind]
+            "insert into names values (" + ",".join(["?"] * 4) + ")", rqst[ind]
```

Comment:
Why is 3 changed with 4 here?

Reply:
Oh right it's because we changed the NCBI dumps files that we are using

Reply:
Actually, I meant to ask about that! I think your DB builder script dropped `unique_names` from the `names` table, but  it looks like it was then retained without 3 being changed to 4?

https://github.com/ImperialCollegeLondon/safedata_validator/blob/c4f6358e394bb8415edc2b8d1147519840118fe2/safedata_validator/taxondb.py#L455